modules/Sources/Utils/SensitiveData.swift

```diff
@@ -82,7 +82,7 @@ extension BlockHeight {
     public var redacted: RedactableBlockHeight { RedactableBlockHeight(self) }
 }
 
-// MARK: - Redactable AccountBalance & SynchronizerState
+// MARK: - Redactable AccountBalance
 
 /// Redactable holder for a block height.
 public struct RedactableAccountBalance: Equatable, Redactable {
```

Comment:
I don't see where the `redacted` method is actually implemented for this type? Is it just the `Undescribable` bound that results it in not being made visible?

Reply:
Yes, Underscribable is defined in the SDK so any object conforming to this protocol gets the redacted behaviours.

```swift
public protocol Undescribable: CustomStringConvertible, CustomDebugStringConvertible, CustomLeafReflectable {}

extension Undescribable {
    public var description: String {
        return "--redacted--"
    }
    
    public var debugDescription: String {
        return "--redacted--"
    }
    
    public var customMirror: Mirror {
        return Mirror(reflecting: "--redacted--")
    }
}
```